content/app/development/api/instance/_index.en.md

```diff
@@ -3,71 +3,70 @@ title: Instance
 linktitle: Instance
 description: How to make changes to application instances.
 toc: true
-tags: [translate-to-english]
 ---
 
-Denne siden er foreløpig ikke fullstendig, mer informasjon vil komme på et senere tidspunkt.
+This page is currently incomplete, more information will be provided at a later date.
 
-En instansiert applikasjon vil ha et tilhørende instans-objekt. Dette objektet inneholder metadata om den spesifikke instansen.
-Om du ønsker å lære mer om instans og api'et rundt så kan du lese teknisk dokumentasjon om dette under API.
+An instantiated application will have a corresponding instance object. This object contains metadata about that specific instance.
+If you want to learn more about instance and its surrounding API you can read the technical documentation for [API](../../api).
 
 ## Substatus
 
-Som app-eier kan man sette en substatus på instansen, dette for å kunne gi sluttbruker ytterligere informasjon om hvilken tilstand instansen befinner seg i.
-Substatus vil vises frem både i meldingsboksen i Altinn og på kvitteringssiden.
+As an app owner you can set a substatus for the instance, this is to enable giving out further information about the condition the instance is currently in.
```

Comment:
```suggestion
As an app owner you can set a substatus for the instance, this is to enable providing more detailed information information about the current state of the instance.
```

Reply:
Seems like you added one too many `information` in your suggestion there, @acn-sbuad